bagger/src/tech_md.py

```diff
@@ -92,3 +98,39 @@ def remodel_file_technical_metadata(root, id_map):
             raise UnrecognisedPRONOMFormatError(format_name.text)
 
         remove_first_child(xmldata)
+
+
+def append_poster(root, destination):
+    # Add this in as an additional file in technical metadata, but not in
+    # structMaps (yet), as that would be just inventing something that needs to
+    # go in Archivematica
+
+    # data we are interested in
+    head, tail = os.path.split(destination)
+    file_name = tail
+    file_size = os.path.getsize(destination)
+    width, height = image_size.get_image_size(destination)
+
+    amds = root.find("./mets:amdSec[@ID='AMD']", namespaces)
```

Comment:
Breaking things up a bit here would make this easier to read.

Reply:
Seconded. Plus a brief comment about what it’s doing to the METS structure – what XML are you trying to create – would be helpful.